src/state/thunks/fetchMappedSet.ts

```diff
@@ -52,22 +61,49 @@ export function fetchMappedSet(visualId: VisualId, mappingId: string, date: Mome
       const [data, geojson] = await Promise.all([
```

Comment:
We could consider extracting the two async arrow functions inside the Promise.all into their separate named functions for better readability. (Either passing the common variables from the state or getting them separately directly from the state.) But the functionality looks good.

Reply:
Updated... check again.